src/Datasource/EntityTrait.php

```diff
@@ -394,7 +394,7 @@ public function isEmpty(string $field): bool
                 empty($value) ||
                 (
                     is_string($value) &&
-                    empty($value)
+                    strlen($value) === 0
```

Comment:
I wonder if with longer strings and lots of ops it would be a quicker nano-opt. to use `!== '0'` instead on top.
Lets see what others think.

Reply:
Given that there's already a `is_string()` check first, `!== ''` check would be better.